src/Keboola/StorageApi/Options/ListFilesOptions.php

```diff
@@ -160,4 +164,22 @@ public function setRunId($runId)
         $this->runId = $runId;
         return $this;
     }
+
+    /**
+     * @return bool
+     */
+    public function showExpired()
```

Comment:
```suggestion
    public function getShowExpired()
```

Reply:
get pro bool? Možná `isShowExpired` ale pro bool mě dává smysl ten showExpired.

Reply:
No koukal jsem jak to maji  udelane ostatni `Options`

Reply:
Máš recht je to jinde jako get upravil sem.